scripts/doc-generation/generate-docs.sh

```diff
@@ -37,11 +37,65 @@ function finish { >&2 echo -e "\033[1m\033[32m✔ $@\033[0m"; }
 
 SCRIPT_DIR=`dirname "$0"`
 
-# Search for Jazzy and install documentation dependencies if necessary.
-if [ -z `which jazzy` ]; then
-    $SCRIPT_DIR/install-documentation-dependencies.sh
+# Install documentation dependencies
+
+# VARIABLES
+COCOAPODS_VERSION="1.7.5"
+JAZZY_VERSION="0.13.6"
+CIRCLECI=${CIRCLECI:-false}
+
+## Install CocoaPods
+if [[ -z `which pod` || $(pod --version) != "${COCOAPODS_VERSION}" ]]; then
```

Comment:
why is cocoapods needed for docs generation?

Reply:
hmm, i can't see a reason. I'll remove it